monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+from monitoring.monitorlib.typing import ImplicitDict
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
+import datetime
+
+class SCDTestIssue(ImplicitDict):
+    ''' A class to hold a message that the test executor can provide to the USS in cases when the USS provides a response that is not same as the expected result. '''
+    timestamp: Optional[str]
+
+    subject: Optional[str]
+    ''' Identifier of the subject of this issue, if applicable. This may be a UAS serial number, or any field of other object central to the issue. '''
+
+    summary: str
+    '''Human-readable summary of the issue'''
+
+    details: str
+    '''Human-readable description of the issue'''
+
+    def __init__(self, **kwargs):
+        super(SCDTestIssue, self).__init__(**kwargs)
+        if 'timestamp' not in kwargs:
+            self.timestamp = datetime.datetime.utcnow().isoformat()
+
+class RequiredResult(ImplicitDict):
+    ''' A class to evaluate results / response to an injection of test flight data (TestFlightRequest) '''
+    expected_result: str
+    incorrect_result_details: Optional[SCDTestIssue]
+
+class TestInjectionRequiredResult(ImplicitDict):
```

Comment:
How will test executor know which of the USSs under test to attempt this injection on?

Reply:
Thanks for this I will add configuration class for your review.

Reply:
The thing I was trying to get at here was that the simulator will probably generate scenarios where injection 1 should go to USS 1, injection 2 to USS 2, injection 3 to USS 1, etc -- the simulator can't know the actual identities of USS 1, USS 2, etc, nor their URLs (which would be defined in a configuration file for a specific test run), but it probably needs a way to tell test executor to which USS out of a supported group of USSs to send each particular injection.

Reply:
Thanks! I have now modified the data structure to have a `sequence_number`, we can change it as necessary,